tests/unittest/llmapi/apps/_test_trtllm_serve_profile.py

```
@@ -0,0 +1,206 @@
+    server.generator.start_profile.assert_called_once_with(
+        output_dir=None,
+        num_steps=None,
+        start_step=0,
+        activities=["CPU", "GPU"],
+    )
+
+
+def test_start_profile_custom_args(server_with_mock_generator):
+    server = server_with_mock_generator
+    request = StartProfileRequest(
+        output_dir="/tmp/my_traces",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/my_traces"

(S108)